### ISSUE TITLE
Links on website missing context

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-design-system#4023]
This issue is from the accessibility audit of the Design System website by DAC in July 2024.

## DAC's description

* Page: [Accessibility strategy – GOV.UK Design System](https://design-system.service.gov.uk/accessibility/accessibility-strategy/)
* WCAG reference: none (Usability)
* Issue ID: DAC_Associated_Text_Usability_01

There were a list of links present that had the associated date present as to when the research was undertaken. The information in relation to the date was not included within the programmatical link and associated via text. Including this content within the programmatical link will provide more information to screen reader users when viewing the page out of context.

<img width="565" alt="Screenshot of links with a year afterwards" src="https://github.com/user-attachments/assets/4dac5981-ce11-41de-b6de-959d4084f504">

## DAC's proposed solution

We suggest including the associated date within the link text. As a result of this, it will provide screen readers with more information when viewing the page out of context.

## Disagreement

I don't think the date is what's important here, it's more important to give context to the link, for example saying it's a blog post or even repeating its title, or mentioning it's a plan on GitHub, etc.

## Additional instances

As this can potentially happen on any page, this work should include checking all the other pages other links missing context.
It might be worth making two different cards for this: One for checking and recording all instances, the other for fixing them.

To save time, it makes sense to audit all the body copy issues together:
* #4010
* #4013
* #4023
* #4024
* #4025

## Needed roles

Content designer

**Comment by calvin-lau-sig7** [comment]
This one will probably tackled alongside #4013. In the specific instance reviewed by DAC, we'll probably write new link text.

**Comment by calvin-lau-sig7** [comment]
Resolved by #4267
